dashboard/test/helpers/plc_evaluation_helper_test.rb

```diff
@@ -0,0 +1,120 @@
+    @evaluation_multi2 = EvaluationMulti.create_from_level_builder({name: 'Question 2'}, {dsl_text: q2_dsl})
+    @evaluation_multi3 = EvaluationMulti.create_from_level_builder({name: 'Question 3'}, {dsl_text: q3_dsl})
+    @evaluation_multi4 = EvaluationMulti.create_from_level_builder({name: 'Question 4'}, {dsl_text: q4_dsl})
+    @evaluation_multi5 = EvaluationMulti.create_from_level_builder({name: 'Question 5'}, {dsl_text: q5_dsl})
+
+    levelgroup_dsl = <<DSL
+    name 'Crossing the bridge of death'
+    title 'Bridge of Death Evaluation'
+    submittable 'true'
+
+    page
+    level 'Question 1'
+    level 'Question 2'
+    level 'Question 3'
+    level 'Question 4'
+    level 'Question 5'
+DSL
+
+    @evaluation = LevelGroup.create_from_level_builder({name: 'evaluation'}, {dsl_text: levelgroup_dsl})
+  end
+
+  test 'submit evaluation enrolls user in appropriate modules' do
+    assert_equal [@module_blue.id, @module_honesty.id], get_preferred_modules_for_answers([[@evaluation_multi1, 'Sir Lancelot'],
+                                                                                           [@evaluation_multi2, 'I seek the grail'],
+                                                                                           [@evaluation_multi3, 'Blue']])
+
+    assert_equal [@module_ignorance.id, @module_no_nickname.id], get_preferred_modules_for_answers([[@evaluation_multi1, 'Sir Robin'],
+                                                                                           [@evaluation_multi2, 'Yes, yes, I seek the Grail'],
+                                                                                           [@evaluation_multi4, 'I dont know that!']])
+
+    assert_equal [@module_cliffs.id, @module_honesty.id], get_preferred_modules_for_answers([[@evaluation_multi1, 'Sir Galahad'],
+                                                                                           [@evaluation_multi2, 'I seek the grail'],
+                                                                                           [@evaluation_multi3, 'Yellow - no, blue']])
+
+    assert_equal [@module_ornithology.id, @module_honesty.id], get_preferred_modules_for_answers([[@evaluation_multi1, 'King Arthur'],
+                                                                                           [@evaluation_multi2, 'I seek the grail'],
+                                                                                           [@evaluation_multi5, 'What do you mean, an African or European Swallow?']])
+
+    assert_equal [], get_preferred_modules_for_answers([[@evaluation_multi1, 'Mr Edgecase'],
+                                                                                           [@evaluation_multi2, 'I seek something else'],
+                                                                                           [@evaluation_multi5, '15 m/s']])
+  end
+
+  def get_preferred_modules_for_answers answers_map
```

Comment:
Nit: missing parens in method definition (1 more above) https://github.com/bbatsov/ruby-style-guide#method-parens